src/commands/cancel.ts

```diff
@@ -0,0 +1,17 @@
+import Command from '../@types/Command'
+
+const cancelShortcut: Command = {
+  id: 'cancel',
+  label: 'Cancel',
+  description: 'Cancel the current gesture.',
+  gesture: undefined,
+  keyboard: { key: ',', meta: true },
+  multicursor: 'ignore',
+  svg: () => null,
+  exec: dispatch => {
+    alert('cancel')
+  },
```

Comment:
If `exec` is not used, you can use a [noop](https://github.com/cybersemics/em/blob/db00102d003112ea1d8cb04993dce3e5f77e7c85/src/constants.ts#L133). We wouldn't want to do `window.alert`.